extensions/community_extensions/deprecation_test_label.md

```diff
@@ -0,0 +1,65 @@
+### Cookie Deprecation: Pass Chrome-facilitated cookie deprecation test label
+
+**Sponsor**: Epsilon
+
+**Background**:  
+Google recently published [an update](https://developer.chrome.com/docs/privacy-sandbox/chrome-testing/) regarding the Chrome-facilitated cookie deprecation testing labels announced in May. Based on the description, Chrome will label ~10 percent of Chrome instances globally. A browser, if labeled, is either Mode A (opt-in testing) or Mode B (1%, forced cookie suppression). Two means are offered to access the label. One can read a request header after setting a special partitioned (CHIPS) cookie or client-side via a new `Navigator` script API promise.
+
+**Proposal**:  
+Entities are encouraged to access the label and to share the value unmodified with partners via the following `Device` extension.
+
+### Object: `Device.ext` <a name="object"></a>
+ <table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>chrometestlabel</code></td>
```

Comment:
My _only_ comment is on naming, so I hesitate to bring this up. :) However, this could be 1. shorter and 2. more specific to the test (I believe this is the value for the `Sec-Cookie-Deprecation` HTTP header).

Reply:
It's a good question. On one hand, it's "just a test". On the other hand, 10% of Chrome is ~7% of web traffic and the bytes add up fast.

If we could expect to use many of these labels, I'd suggest an object (e.g., `labels`) and then children of that (e.g., `cdep` or `cookiedep`). However, this structure feels like over-engineering.

So, back to a single field. If we could quickly settle on something shorter, like `cdep` or even `cookiedep`, that would be helpful. I know this is supposed to be temporary and I want to say I won't lose a lot of sleep over a longer name. And I don't want to extend this conversation more than necessary -- we just need a name so we can move forward. But things tend to hang around longer than expected, and we do prefer short names in the spirit of sustainability.

TL;DR the shorter the better, and we use the spec to explain the field. If you're good to go with `pslabel` that seems reasonable to me, and thanks for the discussion.

Reply:
>If we could expect to use many of these labels, I'd suggest an object (e.g., labels) and then children of that (e.g., cdep or cookiedep). However, this structure feels like over-engineering.

Agree, @simontrasler. There is supposed to be but one label per browser. Either some flavor a percentage gradient in Mode A, Mode B or, potentially, a label indicating "not A or B," as our friends at Criteo have [suggested](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/issues/153) to Google support. Please +1 that if you agree.

>So, back to a single field. If we could quickly settle on something shorter, like cdep or even cookiedep, that would be helpful....we do prefer short names in the spirit of sustainability

I opened with `chrometestlabel` as a conversation starter. Mission accomplished. 
SOLD to the man proposing `cdep`! 

Anyone else? Bueller? Bueller?

Reply:
Updated to `cdep`